proxy/http/HttpTunnel.cc

```diff
@@ -1151,6 +1151,10 @@ HttpTunnel::producer_handler(int event, HttpTunnelProducer *p)
 
   // Handle chunking/dechunking/chunked-passthrough if necessary.
   if (p->do_chunking) {
+    if (p->chunked_handler.chunked_buffer && p->chunked_handler.chunked_buffer->max_read_avail() > 524288) {
```

Comment:
Why didn't you define a constant for this? It's the same issue with the H2 issue, right?

Reply:
Yes, it is a similar issue.  I can do that.

Reply:
Yeh, +1 on at least having one central define for this (instead of a configuration option).